cmd/web/web.go

```diff
@@ -26,6 +36,22 @@ func NewWebCmd() *cobra.Command {
 	serveCmd.Flags().IntVarP(&port, "port", "p", 8080, "The port for the HTTP service to listen at")
 	serveCmd.Flags().StringVar(&araAddr, "ara-addr", "127.0.0.1:8000", "Address where ARA is running (ex: localhost:80)")
 
+	serveCmd.Flags().StringVar(&dbhost, "dbhost", "localhost", "The database host")
+	serveCmd.Flags().StringVar(&dbport, "dbport", "5432", "The database port to connect to")
+	serveCmd.Flags().StringVar(&dbuser, "dbuser", "postgres", "The database user")
+	serveCmd.Flags().StringVar(&dbpassword, "dbpassword", "postgres", "The database password")
+	serveCmd.Flags().StringVar(&dbname, "dbname", "trento", "The database name that the application will usee")
+
+	// Bind the flags to viper and make them available to the application
+	serveCmd.Flags().VisitAll(func(f *pflag.Flag) {
+		// Needed if the flags contain dashes
+		if strings.Contains(f.Name, "-") {
+			viper.BindEnv(f.Name, strings.ToUpper(strings.ReplaceAll(f.Name, "-", "_")))
+		}
+
+		viper.BindPFlag(f.Name, f)
```

Comment:
As a question. In the flags where we have the `-`, the name is uppercased, but here it is not.
Is this done by viper? Or it is just forgotten?

Reply:
Viper skips all the names that are considerd invalid in the `AutoEnv()` method called by `root.cmd` at the end of the command setup.
 
`ara-addr` is one of them so you need to call manually `BindEnv` , otherwise it will not work.

Reply:
EDIT: we found a way to improve it and make this bnehaviour automatic for all the "dashed" commands, just fixup/squashed it